packages/features/ee/teams/components/TeamAvailabilityScreen.tsx

```diff
@@ -33,86 +33,87 @@ export default function TeamAvailabilityScreen(props: Props) {
     const member = props.team?.members?.[index];
     if (!member) return <></>;
 
-    return (
-      <div key={member.id} style={style} className="flex border-r border-gray-200 pl-4 ">
-        <TeamAvailabilityTimes
-          teamId={props.team?.id as number}
-          memberId={member.id}
-          frequency={frequency}
-          selectedDate={selectedDate}
-          selectedTimeZone={selectedTimeZone}
-          HeaderComponent={
-            <div className="mb-6 flex items-center">
-              <Avatar
-                size="sm"
-                imageSrc={CAL_URL + "/" + member.username + "/avatar.png"}
-                alt={member?.name || ""}
-                className="min-w-10 min-h-10 mt-1 h-10 w-10 rounded-full"
-              />
-              <div className="ml-3 inline-block overflow-hidden pt-1">
-                <span className="truncate text-lg font-bold text-neutral-700">{member?.name}</span>
-                <span className="-mt-1 block truncate text-sm text-gray-400">{member?.email}</span>
-              </div>
-            </div>
-          }
-        />
-      </div>
-    );
-  };
+    //   return (
+    //     <div key={member.id} style={style} className="flex border-r border-gray-200 pl-4 ">
```

Comment:
Why?

Reply:
Also not being used in V2 and was causing type errors